src/App/MinerInfo/MinerInfo.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './MinerInfo.css';
+
+export const MinerInfo = (props) => {
```

Comment:
if you want to destructure and use your props you can actually do that in parameter. It would look like:

+export const MinerInfo = ({title, data}) => {